### PR TITLE
docs(samples): add AutoML image classification sample

### PR DESCRIPTION
**Pull request by andrewferlitsch**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [x] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-aiplatform/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #202042079🦕

### CODE REVIEW

**Comment by andrewferlitsch** [comment]
The build is failing:

import tensorflow as tf
E   ModuleNotFoundError: No module named 'tensorflow'

That seems to be part of the testing pipeline, so not sure how to fix.

**Comment by morgandu** [comment]
Please change the PR title from "fix: ..." to "samples: ...".

"fix: ..." and "feat: ..." are reserved for Vertex SDK and the title content will be propagated to the Release Note.

**Review by morgandu — CHANGES_REQUESTED** [review]
Please change the PR title from "fix: ..." to "samples: ...".

"fix: ..." and "feat: ..." are reserved for Vertex SDK and the title content will be propagated to the Release Note.

**Comment by kweinmeister** [comment]
Also, to add to @morgandu's comment, it's preferred if we don't refer to internal bug numbers that some users may not have access to.

So, could the title also be changed to include a summary of what was actually fixed? That will make the Release Notes for this fix more useful.

File: samples/model-builder/predict_image_classification_sample.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+from typing import List
+
+from google.cloud import aiplatform
+
+import tensorflow as tf
+
+
+#  [START aiplatform_sdk_predict_image_classification_sample]
+def predict_tabular_classification_sample(
```
Comment (kweinmeister) [inline samples/model-builder/predict_image_classification_sample.py:26]: Should this read `predict_image_classification_sample`?

Comment (andrewferlitsch) [reply samples/model-builder/predict_image_classification_sample.py:26]: yes, that looks like a typo.

Comment (andrewferlitsch) [reply samples/model-builder/predict_image_classification_sample.py]: I fixed it.

**Review by kweinmeister — APPROVED** [review]

**Review by morgandu — APPROVED** [review]

**Pull request merged**